src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -412,21 +428,20 @@ internal Location CreateLocation(AnnotatedCodeLocationVersionOne v1AnnotatedCode
                 Message = CreateMessage(message)
             };
 
-            LogicalLocation logicalLocation;
+            LogicalLocation logicalLocation = new LogicalLocation();
 
-            if (!string.IsNullOrWhiteSpace(logicalLocationKey) &&
-                _currentRun.LogicalLocations.TryGetValue(logicalLocationKey, out logicalLocation))
-            {
-                logicalLocation.FullyQualifiedName = fullyQualifiedLogicalName;
-                logicalLocation.Name = GetLogicalLocationName(fullyQualifiedLogicalName);
-                location.FullyQualifiedLogicalName = logicalLocationKey;
-            }
-            else if (!string.IsNullOrWhiteSpace(fullyQualifiedLogicalName))
+            logicalLocation.FullyQualifiedName = fullyQualifiedLogicalName;
+            logicalLocation.Name = GetLogicalLocationName(fullyQualifiedLogicalName);
+            location.FullyQualifiedLogicalName = logicalLocationKey;
```

Comment:
>logicalLocationKey [](start = 49, length = 18)

Again, the key/FQLN reversal.

Reply:
covered elsewhere

---
In reply to: [243868533](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243868533) [](ancestors = 243868533)